shared/src/commonMain/kotlin/com/mbta/tid/mbta_app/viewModel/FavoritesViewModel.kt

```diff
@@ -0,0 +1,186 @@
+package com.mbta.tid.mbta_app.viewModel
+
+import androidx.compose.runtime.Composable
+import androidx.compose.runtime.LaunchedEffect
+import androidx.compose.runtime.getValue
+import androidx.compose.runtime.mutableStateOf
+import androidx.compose.runtime.remember
+import androidx.compose.runtime.setValue
+import co.touchlab.skie.configuration.annotations.DefaultArgumentInterop
+import com.mbta.tid.mbta_app.model.RouteCardData
+import com.mbta.tid.mbta_app.model.RouteStopDirection
+import com.mbta.tid.mbta_app.model.response.AlertsStreamDataResponse
+import com.mbta.tid.mbta_app.model.response.GlobalResponse
+import com.mbta.tid.mbta_app.usecases.FavoritesUsecases
+import io.github.dellisd.spatialk.geojson.Position
+import kotlinx.coroutines.flow.Flow
+import kotlinx.datetime.Clock
+import kotlinx.datetime.Instant
+
+class FavoritesViewModel(private val favoritesUsecases: FavoritesUsecases) :
```

Comment:
question (non-blocking): Thinking about the intersection of adopting Molecule and some of the learnings from https://github.com/mbta/mobile_app/pull/1064. What do you think of making an interface for FavoritesViewModel so that it is easier to mock the fireEvent wrappers functions for UI tests?